lib/ansible/modules/crypto/luks_device.py

```diff
@@ -193,10 +231,41 @@ class Handler(object):
     def __init__(self, module):
         self._module = module
         self._lsblk_bin = self._module.get_bin_path('lsblk', True)
+        self._blkid_bin = self._module.get_bin_path('blkid', True)
```

Comment:
`blkid` is not always installed (for example Debian 9 and 10 don't necessarily include it). How about not requiring it (i.e. use `False`), and checking whether `self._blkid_bin` contains a result when you actually need it, and if then it doesn't, fail the module? Then `blkid` is not a requirement to run the module at all (right now it is).

Reply:
You can also move this line into `get_device_by_uuid()` and `get_device_by_label()`; then Ansible will fail for you if it cannot find the command, and it will only search for it when we actually need it.

Reply:
Hmmm, I choose the second option, thanks for the tip!